buildSrc/src/main/rocker/com/uber/okbuck/template/android/ManifestRule.rocker.raw

```diff
@@ -14,14 +14,16 @@ okbuck_manifest(
     main_manifest = '@mainManifest',
     min_sdk = '@minSdk',
     target_sdk = '@targetSdk',
+@if (valid(pkg)) {
+    package = '@pkg',
+} else {
+    package = '',
```

Comment:
probably can skip passing package. does it replace the package with an empty string?

Reply:
Done